### PR TITLE
Add Stream type

### PR DESCRIPTION
**Pull request by twoeths**

### CODE REVIEW

**Review by wemeetagain — APPROVED** [review]
:+1:

File: types/interface-connection/index.d.ts [hunk 1]
```diff
@@ -5,11 +5,22 @@
 
 /// <reference types="multiaddr"/>
 
+declare interface Stream {
+    source: Function;
+    sink: Function;
```
Comment (mpetrunic) [inline types/interface-connection/index.d.ts:10]:
```
  sink(source: Iterable<unknown>);
  source: Record<string, unknown>;
```
these might be better types

**Review by mpetrunic — COMMENTED** [review]
A bit too late but...

File: types/interface-connection/index.d.ts (hunk 1 above)
Comment (twoeths) [reply types/interface-connection/index.d.ts:10]: I'll see if I have other PRs I'll include it,
or I'll include it separately.

Comment (mpetrunic) [reply types/interface-connection/index.d.ts:10]: I think there is also AsyncIterator type, would be nice to typehint it properly

**Pull request merged**